L1Trigger/TrackFindingTMTT/interface/KFTrackletTrack.h

```diff
@@ -56,11 +61,11 @@ namespace tmtt {
           tanLambda_(tanLambda),
           chi2rphi_(chi2rphi),
           chi2rz_(chi2rz),
-          done_bcon_(false),
-          qOverPt_bcon_(qOverPt),
-          d0_bcon_(d0),
-          phi0_bcon_(phi0),
-          chi2rphi_bcon_(chi2rphi),
+          done_bcon_(done_bcon),
```

Comment:
If we choose to add the "bcon" arguments to the KFTrackletTrack constructor, then the setBeamConst() function is no longer needed. Or if we choose to keep the setBeamConstr() function, then the "bcon" arguments don't need adding to the constructor.

Reply:
What do you prefer?

Reply:
Either solution is fine. Choose whichever is easiest for you.

Reply:
I took out the setBeamConstr() function